src/SampleWebApp/Application/Behavior/GetStoresQuerySecurityFilter.cs

```diff
@@ -10,7 +11,7 @@ public async Task<IQueryable<StoreDto>> Handle(GetStoresQuery request, RequestHa
         {
             var result = await next();
 
-            return result.Where(store => store.Id == "EUR");
+            return result.Where(store => store.Id == Nuid.From("Shippi Tivoli Einkaufscenter.Shopping Center, 8957 Spreitenbach, CH").Value);
```

Comment:
Lets come up with a better constraint example. this is unmaintainable and unrealistic.